Python/pywarpx/fields.py

```diff
@@ -508,6 +508,20 @@ def _setitem2d(self, index, value):
                     fields[i][sss] = value
 
 
+def rhoWrapper(level=0, include_ghosts=False):
+    return _MultiFABWrapper(direction=None,
+                            get_lovects=_libwarpx.get_mesh_charge_density_fp_lovects,
+                            get_fabs=_libwarpx.get_mesh_charge_density_fp,
+                            get_nodal_flag=_libwarpx.get_Rho_nodal_flag,
+                            level=level, include_ghosts=include_ghosts)
+
+def phiWrapper(level=0, include_ghosts=False):
```

Comment:
This should be renamed `PhiFPWrapper` to be consistent with the other wrappers. Also, this should be moved to just after `RhoFPWrapper`.